modules/nf-core/deseq2/differential/main.nf

```diff
@@ -0,0 +1,30 @@
+process DESEQ2_DIFFERENTIAL {
+    tag "$contrast_meta"
+    label 'process_medium'
+
+    conda (params.enable_conda ? "bioconda::bioconductor-deseq2=1.32.0" : null)
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/bioconductor-deseq2%3A1.32.0--r41h399db7b_0' :
+        'quay.io/biocontainers/bioconductor-deseq2:1.32.0--r41h399db7b_0' }"
+
+    input:
+    path samplesheet
+    path counts
+    val contrast_meta
+
+    output:
+    tuple val(contrast_meta), path("deseq2.results.tsv")               , emit: results
```

Comment:
Do we want a prefix on the output files? Currently every time you run this module, all the output files will be identically named. Which means that if you want to do any downstream combining of the outputs or just outputting them into a folder, you'll have to ensure they are renamed or put into a folder in some way anyway.

Reply:
Haha, I actually had prefixes at one point derived from concatenating the contrast meta. I removed it in favour of a `publishDir` doing the right thing from the workflow, but I can reinstate the prefix.